js/core.js

```diff
@@ -152,7 +152,7 @@ window.onload = async () => {
     console.log("Loaded Quest");
 
     $("#quest_version").html(Quest.tag_name.replace("v", ""));
-    $("#quest_download_btn").attr("href", Quest.assets[0].browser_download_url);
+    $("#quest_download_btn").attr("href", Quest.assets[1].browser_download_url);
```

Comment:
This is perfectly fine. [0] retrieves the first release in the array, which is the latest release.

Arrays are index starting at 0, not 1. Change this please.